bin/check-mail-delay.rb

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env ruby
+#
+# Check for mail delays in the Postfix mail queue. This is split out into its
+# own check because it requires storing details about every message in the
+# queue in memory, which may not be desirable on heavily-trafficked systems.
+
+# ===
+#
+# Copyright (c) 2016, Jonathan Hartman <j@hartman.io>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+#
+
```

Comment:
No problem. Wasn't sure if you'd want it to match the style that was already there or the one from the developer guide. Done.